pkgs/tools/networking/shadowfox/default.nix

```diff
@@ -0,0 +1,18 @@
+{ fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "shadowfox-updater-${version}";
```

Comment:
The package in `all-packages.nix` is called `shadowfox`. It should have the same name here to avoid confusion (not `shadowfox-updater`).